megamek/src/megamek/common/EntityWeightClass.java

```diff
@@ -252,8 +250,6 @@ public static int getWeightClass(double tonnage, Entity en) {
                     break;
                 }
             }
-        } else if (en instanceof ConvFighter) {
```

Comment:
In the other method, you added conventional fighters to the aero else if, here, they're just removed?

Reply:
ConvFighter is instanceof Aero so the Aero part works for ConvFighters in the second method.